### DISCUSSION TITLE
Advice for passing CLI options as the value of a command option

### DISCUSSION THREAD

**Discussion by fossbrandon** [Tyrrrz/CliFx#136] [Q&A] [answered]
Hi there, I'm currently creating a program that wraps various other CLI programs. Some of the options I'm trying to create are to override all the arguments passed to these tools instead of using the convenience options that I've created for them (aiming for more fine-tuned control if they want it without my command becoming too exhaustive or tool specific). My first attempt at this was to take this input as a string since that seems simplest. Ex: `dotnet example-program --override-tool-args "--check"`. However, this results in this error:

```
Unrecognized option(s):
--check
```

So, I'm looking for advice on either a better approach to my needs, or if this type of functionality could be considered as a new or updated feature of this library.

**Comment by Tyrrrz** [comment, marked as answer]
There is no nice way to do this currently. Issue #39 is related to this.

**Reply by fossbrandon** [reply]
Thanks, I didn't see that issue. I'll try to rethink my application to work around this.

**Reply by Tyrrrz** [reply]
In your case, I believe you can workaround this by prepending a space inside the quotes, i.e. `--override-tool-args " --check"`. That should prevent it from being treated as a separate option.
